georiviere/observations/filters.py

```diff
@@ -23,8 +21,8 @@ class StationFilterSet(WatershedFilterSet, ZoningFilterSet, MapEntityFilterSet):
         queryset=Parameter.objects.all(),
     )
     in_service = ChoiceFilter(
+        empty_label=_('In service'),
```

Comment:
is translation already registred in po files ?

Reply:
It is ! https://github.com/Georiviere/Georiviere-admin/blob/master/georiviere/observations/locale/fr/LC_MESSAGES/django.po#L103